### DISCUSSION TITLE
[SPAM] I wanted to thank the creators of pester

### DISCUSSION THREAD

**Discussion by slider56** [pester/Pester#2159] [General]
I was so lucky enough to lose several hundred scientific papers and a number of written works , rare applications and peace of mind. I was tormented by the "ThoughtfulInstaller" for 2 days until I could find the program. It took hours to delete all the program files because of all the restrictions the placed on them.  Your Code of Conduct is ineffective. While I understand this is not your fault, a program thtt is freely available that can impair virtuallly every computer function. If you like details please contact me.

**Comment by nohwnd** [comment]
Reported as spam, including link to this. Will delete later once the user is blocked.
